quadlet/localstack/localstack.container

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=A Localstack container for local development of AWS services
+After=network-online.target
+
+[Container]
+Image=docker.io/localstack/localstack:latest
+AutoUpdate=registry
+ContainerName=localstack
+Environment=DOCKER_SOCK=unix:///var/run/podman/podman.sock
+Environment=DOCKER_HOST=unix:///var/run/podman/podman.sock
+Environment=DOCKER_CMD=podman
+PublishPort=4566:4566
+PublishPort=4510-4559:4510-4559
+Volume=localstack.volume:/var/lib/localstack:Z
+Volume=/var/run/user/%G/podman/podman.sock:/var/run/podman/podman.sock:ro
```

Comment:
AFAIK (and true in my Fedora 39) `/var/run` is a sym-link to `/run`. So, consider changing the host path

Reply:
Good spot.  This should hopefully make the quadlets more compatible with other distros.

Updated all occurrences of `/var/run` (inside and outside the container) to `/run`